THIRD-PARTY-NOTICES

```diff
@@ -31,7 +22,7 @@ Developed by:
     http://llvm.org
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of
-this software and associated documentation files (the "Software"), to deal with
+this software and associated documentation files (the ""Software""), to deal with
```

Comment:
Why are the quotes duplicated? They are not duplicated in CoreCLR version of this file.